mmcls/datasets/cub.py

```diff
@@ -0,0 +1,123 @@
+        'Orchard_Oriole', 'Scott_Oriole', 'Ovenbird', 'Brown_Pelican',
+        'White_Pelican', 'Western_Wood_Pewee', 'Sayornis', 'American_Pipit',
+        'Whip_poor_Will', 'Horned_Puffin', 'Common_Raven',
+        'White_necked_Raven', 'American_Redstart', 'Geococcyx',
+        'Loggerhead_Shrike', 'Great_Grey_Shrike', 'Baird_Sparrow',
+        'Black_throated_Sparrow', 'Brewer_Sparrow', 'Chipping_Sparrow',
+        'Clay_colored_Sparrow', 'House_Sparrow', 'Field_Sparrow',
+        'Fox_Sparrow', 'Grasshopper_Sparrow', 'Harris_Sparrow',
+        'Henslow_Sparrow', 'Le_Conte_Sparrow', 'Lincoln_Sparrow',
+        'Nelson_Sharp_tailed_Sparrow', 'Savannah_Sparrow', 'Seaside_Sparrow',
+        'Song_Sparrow', 'Tree_Sparrow', 'Vesper_Sparrow',
+        'White_crowned_Sparrow', 'White_throated_Sparrow',
+        'Cape_Glossy_Starling', 'Bank_Swallow', 'Barn_Swallow',
+        'Cliff_Swallow', 'Tree_Swallow', 'Scarlet_Tanager', 'Summer_Tanager',
+        'Artic_Tern', 'Black_Tern', 'Caspian_Tern', 'Common_Tern',
+        'Elegant_Tern', 'Forsters_Tern', 'Least_Tern', 'Green_tailed_Towhee',
+        'Brown_Thrasher', 'Sage_Thrasher', 'Black_capped_Vireo',
+        'Blue_headed_Vireo', 'Philadelphia_Vireo', 'Red_eyed_Vireo',
+        'Warbling_Vireo', 'White_eyed_Vireo', 'Yellow_throated_Vireo',
+        'Bay_breasted_Warbler', 'Black_and_white_Warbler',
+        'Black_throated_Blue_Warbler', 'Blue_winged_Warbler', 'Canada_Warbler',
+        'Cape_May_Warbler', 'Cerulean_Warbler', 'Chestnut_sided_Warbler',
+        'Golden_winged_Warbler', 'Hooded_Warbler', 'Kentucky_Warbler',
+        'Magnolia_Warbler', 'Mourning_Warbler', 'Myrtle_Warbler',
+        'Nashville_Warbler', 'Orange_crowned_Warbler', 'Palm_Warbler',
+        'Pine_Warbler', 'Prairie_Warbler', 'Prothonotary_Warbler',
+        'Swainson_Warbler', 'Tennessee_Warbler', 'Wilson_Warbler',
+        'Worm_eating_Warbler', 'Yellow_Warbler', 'Northern_Waterthrush',
+        'Louisiana_Waterthrush', 'Bohemian_Waxwing', 'Cedar_Waxwing',
+        'American_Three_toed_Woodpecker', 'Pileated_Woodpecker',
+        'Red_bellied_Woodpecker', 'Red_cockaded_Woodpecker',
+        'Red_headed_Woodpecker', 'Downy_Woodpecker', 'Bewick_Wren',
+        'Cactus_Wren', 'Carolina_Wren', 'House_Wren', 'Marsh_Wren',
+        'Rock_Wren', 'Winter_Wren', 'Common_Yellowthroat'
+    ]
+
+    def __init__(self, *args, ann_file, image_class_labels_file,
+                 train_test_split_file, **kwargs):
+        self.image_class_labels_file = image_class_labels_file
+        self.train_test_split_file = train_test_split_file
+        super(CUB, self).__init__(*args, ann_file=ann_file, **kwargs)
+
+    def load_annotations(self):
+        with open(self.ann_file) as f:
+            samples = [x.strip().split(' ')[1] for x in f.readlines()][:100]
+
+        with open(self.image_class_labels_file) as f:
+            gt_labels = [int(x.strip().split(' ')[1])
```

Comment:
```suggestion
            gt_labels = [int(x.strip().split(' ')[1]) - 1
```

the original labels are from 1 to 200, training the CUB full dataset will get an error.: https://stackoverflow.com/questions/51691563/cuda-runtime-error-59-device-side-assert-triggered.